VotingApplication/VotingApplication.Web/Scripts/Services/ManageService.js

```diff
@@ -49,6 +49,24 @@
             });
         };
 
+        self.getPollType = function (manageId) {
+
+            if (!manageId) {
+                return null;
```

Comment:
Should this not return a rejected promise, in keeping with the other return values?